airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/orchestration/legacy_typing_deduping/TypingDedupingWriter.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.orchestration.legacy_typing_deduping
+
+import io.airbyte.cdk.load.command.DestinationStream
+import io.airbyte.cdk.load.orchestration.ColumnNameMapping
+import io.airbyte.cdk.load.orchestration.DestinationHandler
+import io.airbyte.cdk.load.orchestration.DestinationInitialStatusGatherer
+import io.airbyte.cdk.load.orchestration.TableNames
+import io.airbyte.cdk.load.write.DestinationWriter
+import io.airbyte.cdk.load.write.StreamLoader
+import java.util.concurrent.Executors
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+
```

Comment:
taken from DefaultSyncOperation. I killed the migration stuff - I don't think the old CDK's implementation was particularly good, and we're killing all our migrations anyway.